tests/test_mpscircuit.py

```diff
@@ -1,13 +1,17 @@
 # pylint: disable=unused-variable
```

Comment:
suggest testing some basic AD and jit support in this file, too

Reply:
How about doing this in the next PR? AD and jit is basically not considered in this PR.